eq/deflect/eventHandler.cpp

```diff
@@ -36,6 +36,56 @@ namespace
 {
 typedef std::vector< EventHandler* > EventHandlers;
 static lunchbox::PerThread< EventHandlers > _eventHandlers;
+
+// Values come from QtCore/qnamespace.h, but don't want to depend on Qt just for that
```

Comment:
Well, for a user who is only interested in a few key events (like ESC) AND does not use Qt the symbolic names would be handy.